<launch>
    <master auto="start"/>

    <!-- start optitrack -->
    <include file="$(find optitrack_person)/launch/optitrack.launch"/>

    <!-- set logging level -->
    <!-- env name="ROSCONSOLE_CONFIG_FILE" value="$(find optitrack_person)/launch/custom_rosconsole.conf"/-->

    <!--  add static transform publisher  -->
    <node pkg="tf" type="static_transform_publisher" name="optitrack_link" args="0 0 0 0 0 0 /map /optitrack 100"/>

    <!--  start optitrack_person  -->
    <!--  <param name="topic_base" type="str" value="/optitrack/bodies/person" />  -->
    <!--  <param name="published_topic" type="str" value="tracked_persons" />  -->
    <!--  <param name="optitrack_frame_id" type="str" value="optitrack" />  -->
    <param name="publish_rate" type="int" value="100" />
    <param name="publish_info" type="bool" value="true" />
    <node pkg="optitrack_person" type="optitrack_person_node_full" name="optitrack_person_node_full" output="screen" required="true"/>
</launch>
